config/guiserver.service.template

```diff
@@ -0,0 +1,47 @@
+[Unit]
+Description=GUIServer
+
+[Service]
+Type=simple
+
+Environment={{if https_proxy}}"https_proxy={{https_proxy}}"{{endif}} \
+    {{if http_proxy}}"http_proxy={{http_proxy}}"{{endif}} \
+    {{if no_proxy}}"no_proxy="{{no_proxy}}"{{endif}}
+
+ExecStart=/usr/bin/python /usr/local/bin/runserver.py \
+    --logging="{{builtin_server_logging}}" \
+    --sslpath="{{ssl_cert_path}}" \
+    --charmworldurl="{{charmworld_url}}" \
+    {{if port}}
+        --port={{port}} \
+    {{endif}}
+    {{if sandbox}}
+        --sandbox \
+    {{else}}
+        --apiurl="{{api_url}}" --apiversion="{{api_version}}" \
+    {{endif}}
+    {{if serve_tests}}
+        --testsroot="{{tests_root}}" \
+    {{endif}}
+    {{if insecure}}
+        --insecure \
+    {{endif}}
+    {{if juju_gui_debug}}
+        --jujuguidebug \
+    {{endif}}
+    {{if env_password}}
+        --password="{{env_password}}" \
+    {{endif}}
+    {{if env_uuid}}
+        --uuid="{{env_uuid}}" \
+    {{endif}}
+    {{if juju_version}}
+        --jujuversion="{{juju_version}}" \
+    {{endif}}
+    {{if jem_location}}
+        --jemlocation="{{jem_location}}" \
+    {{endif}}
+    {{if interactive_login}}
+        --interactivelogin="{{interactive_login}}" \
+    {{endif}}
+    .
```

Comment:
If we're now generating two different scripts I have concerns about maintainability--any option we add now has to be updated here *and* in the other script. Maybe we need to revisit how we're passing options to guiserver.